src/secure_random.cr

```diff
@@ -16,13 +16,13 @@ require "base64"
 # ```
 #
 # The implementation follows the
-# [libsodium sysrandom](https://github.com/jedisct1/libsodium/blob/6fad3644b53021fb377ca1207fa6e1ac96d0b131/src/libsodium/randombytes/sysrandom/randombytes_sysrandom.c)
+# [libsodium sysrandom](https://github.com/jedisct1/libsodium/blob/master/src/libsodium/randombytes/sysrandom/randombytes_sysrandom.c)
```

Comment:
Isn't it better to point this to a specific commit as it used to? `master` may break, for example if they rename or move the file.